problems/sumEvenNumbers.js

```diff
@@ -1,16 +1,21 @@
 /** Sum all even numbers in an array
- * 
-* @param {number[]} arr - The input array
-* @returns {number} - The sum of all even numbers
-*
-* ex: sumEvenNumbers([1, 2, 3, 4])
-* returns: 6 (because 2 + 4 = 6)
-*
-* ex: sumEvenNumbers([4, 3, 10])
-* returns: 14
-*/
-function sumEvenNumbers() {
-
+ *
+ * @param {number[]} arr - The input array
+ * @returns {number} - The sum of all even numbers
+ *
+ * ex: sumEvenNumbers([1, 2, 3, 4])
+ * returns: 6 (because 2 + 4 = 6)
+ *
+ * ex: sumEvenNumbers([4, 3, 10])
+ * returns: 14
+ */
+function sumEvenNumbers(arr) {
+  let evenOnly = arr.filter((el) => {
+    return el % 2 === 0;
+  });
+  return evenOnly.reduce((accum, el) => {
+    return accum + el;
+  });
```

Comment:
Cool answer. 
if you're going to use reduce though, you might as well just check if el is even or odd there and update the accum that you're returning.